ToolkitApp/tabs/src/components/sample/RSCList.css

```diff
@@ -2,8 +2,14 @@
     width: 100%;
 }
 
-#RSCTable tr:nth-child(odd){background-color: #e5e5e5;}
+#RSCTable tr:nth-child(odd) {
+    background-color: #e5e5e5;
+}
 
-#RSCTable tr:nth-child(even){background-color: #f2f2f2;}
+#RSCTable tr:nth-child(even) {
+    background-color: #f2f2f2;
+}
 
-#RSCTable tr:hover {background-color: #ddd;}
\ No newline at end of file
+#RSCTable tr:hover {
```

Comment:
fyi there's a convention to not use id in css if you wanted to read more about it 😃  https://github.com/CSSLint/csslint/wiki/Disallow-ids-in-selectors

Reply:
Thanks, I'll check that out!

Reply:
FYI: I'll be cleaning up the CSS in a later PR, as the CSS is very messy